main.py

```diff
@@ -0,0 +1,239 @@
+    window.blit(Meme, (0, 0))
+    window.blit(Fontimg, (25, 0))
+    window.blit(Fontimg2, (53, 400))
+    pygame.display.update()
+
+# Crusading Meme (Press 'c' on keyboard)
+def crusadememe():
+
+    knight = pygame.image.load('knight.png')
+    pygame.display.set_icon(knight)
+    pygame.display.set_caption("Take back Jerusalem")
+
+    # Meme text font, size, anti-aliasing and colour
+    Textfont = pygame.font.SysFont("impact", 45)
+    Fontimg = Textfont.render("When you were supposed to go crusading", 1, white)
+    Textfont2 = pygame.font.SysFont("impact", 50)
+    Fontimg2 = Textfont2.render("but your squad ditches you", 1, white)
+
+    # Loads, draws and updates the meme/display
+    Meme = pygame.image.load("loneknight.png")
+    window.blit(Meme, (0, 0))
+    window.blit(Fontimg, (20, 10))
+    window.blit(Fontimg2, (120, 450))
+    pygame.display.update()
+
+# Homework Meme (Press 'h' on keyboard)
+def homeworkmeme():
+
+    knight = pygame.image.load('shh.png')
+    pygame.display.set_icon(knight)
+    pygame.display.set_caption("pls stop")
+
+    # Meme text font, size, anti-aliasing and colour
+    Textfont = pygame.font.SysFont("impact", 45)
+    Fontimg = Textfont.render("When someone is about to remind", 1, white)
+    Textfont2 = pygame.font.SysFont("impact", 50)
+    Fontimg2 = Textfont2.render("the teacher about homework", 1, white)
+
+    # Loads, draws and updates the meme/display
+    Meme = pygame.image.load("shh.png")
+    window.blit(Meme, (0, 0))
+    window.blit(Fontimg, (11, 10))
+    window.blit(Fontimg2, (32, 415))
+    pygame.display.update()
+
+# set up program loop
+while not image:
+    for event in pygame.event.get():
+        if event.type == pygame.QUIT:
+            image = True
+        if event.type == KEYDOWN and event.key == K_k:
+            window_width = 1280
+            window_height = 875
+            window = pygame.display.set_mode((window_width, window_height))
+            kitchencats()
+        if event.type == KEYDOWN and event.key == K_r:
+            window_width = 1280
+            window_height = 875
+            window = pygame.display.set_mode((window_width, window_height))
+            makered()
+        if event.type == KEYDOWN and event.key == K_g:
+            window_width = 1280
+            window_height = 875
+            window = pygame.display.set_mode((window_width, window_height))
+            makegreen()
+        if event.type == KEYDOWN and event.key == K_b:
+            window_width = 1280
+            window_height = 875
+            window = pygame.display.set_mode((window_width, window_height))
+            makeblue()
+        if event.type == KEYDOWN and event.key == K_n:
+            window_width = 1280
+            window_height = 875
+            window = pygame.display.set_mode((window_width, window_height))
+            negative()
+        if event.type == KEYDOWN and event.key == K_e:
+            window_width = 640
```

Comment:
Why are you using different window sizes? You could improve sophistication by [scaling or letterboxing](http://www.pygame.org/docs/ref/transform.html) your surfaces.